src/config.rs

```diff
@@ -193,9 +193,11 @@ impl Config {
 
 fn clean_path_string(path_string: &str) -> path::PathBuf {
 	let separator_regex = regex::Regex::new(r"\\|/").unwrap();
+	let trailing_regex = regex::Regex::new(r"(/)+$").unwrap();
```

Comment:
There's a few places I have to deal with raw strings:

- What ultimately goes in the DB
- Input that reaches the API
- User input from the config file

I decided to store paths in the DB using the native separator (whichever one I get from filesystem calls), which means the DB can deal with Path/PathBuf and do messy string handling. For the other two, I have no control over what comes in the other two so I think some sanitization is required. Those two code sites are precisely the ones you fixed so I think we should be in the clear from here, since after these gateways everything does use Path/Pathbuf.

Thanks again for the fixes!